samples/github/program.py

```diff
@@ -6,39 +6,33 @@
 
 API details:
 - REST API referene: https://docs.github.com/en/rest
-- Go client API: https://pkg.go.dev/github.com/google/go-github/v57/github
-
-It also demonstrates using a custom builtin function (rand.intn) to generate
-random integer numbers (based on https://pkg.go.dev/math/rand#Rand.Intn).
+- PyGitHub library: https://pygithub.readthedocs.io/en/latest/index.html
 
 This program isn't meant to cover all available functions and events.
 It merely showcases a few illustrative, annotated, reusable examples.
-
-Starlark is a dialect of Python (see https://bazel.build/rules/language).
 """
 
-load("@github", "github_conn")
+import random
+
+from autokitteh.github import github_client
 
 # https://docs.github.com/en/rest/reactions/reactions#about-reactions
 REACTIONS = ["+1", "-1", "laugh", "confused", "heart", "hooray", "rocket", "eyes"]
 
-def on_github_issue_comment(data):
+
+def on_github_issue_comment(event):
     """https://docs.github.com/en/rest/overview/github-event-types#issuecommentevent
 
     Based on the filter in the "autokitteh.yaml" manifest file,
     handle only *new* issue comments in this sample code
     (FYI, the other options are "edited" and "deleted").
 
     Args:
-        data: GitHub event data.
+        event: GitHub event data.
     """
 
-    # Add to each new issue comment a random reaction emoji.
-    # rand.intn: https://pkg.go.dev/math/rand#Rand.Intn.
-    reaction = REACTIONS[rand.intn(len(REACTIONS))]
-    github_conn.create_reaction_for_issue_comment(
-        owner = data.repo.owner.login,
-        repo = data.repo.name,
-        id = data.comment.id,
-        content = reaction,
-    )
+    g = github_client("github_conn")
```

Comment:
This is totally subjective, but I prefer `github` over `g`. `g` is a little too Golang-ish.

Another cosmetic nit: no need for empty lines above and below this line. FYI, the empty line above was originally auto-inserted because of the (now redundant) comment.

Lastly, if Ruff thinks line 36 is too long and breaks it up, split it into 2 lines:

```
repo = ...
issues = ...
```

Reply:
Done